source/common/http/header_map_impl.h

```diff
@@ -345,6 +345,8 @@ class HeaderMapImpl : NonCopyable {
   const uint32_t max_headers_kb_ = UINT32_MAX;
   // This holds the max count of the headers in the HeaderMap.
   const uint32_t max_headers_count_ = UINT32_MAX;
+
+  template <class T> friend class StaticLookupBenchmarker;
```

Comment:
add a comment on what this is for -- e..g for a benchmark to access non-public method `foo()`